pkgs/tools/wayland/wlsunset/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, meson, pkg-config, ninja, wayland
+, wayland-protocols
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wlsunset";
+  version = "0.1.0";
+
+  src = fetchurl {
+    url = "https://git.sr.ht/~kennylevinsen/wlsunset/archive/${version}.tar.gz";
+    sha256 = "0g7mk14hlbwbhq6nqr84452sbgcja3hdxsqf0vws4njhfjgqiv3q";
+  };
+
+  nativeBuildInputs = [ meson pkg-config ninja wayland ];
+  buildInputs = [ wayland wayland-protocols ];
```

Comment:
wayland shouldn't be in both.

Reply:
AFAIK it actually should because `wayland-scanner` is a "build-time" dependency and the `wayland` library is a "run-time" dependency.

Reply:
Is ``wayland-scanner`` part of the wayland package?

Reply:
Yes